packages/app/src/cli/models/extensions/specifications/app_config_app_access.test.ts

```diff
@@ -13,6 +13,7 @@ describe('app_config_app_access', () => {
         access_scopes: {
           scopes: 'read_products,write_products',
           optional_scopes: ['read_customers'],
+          required_scopes: ['write_orders', 'read_inventory'],
```

Comment:
nit: Do we care if the `required_scopes` don't match the `scopes` in these tests?